### ISSUE TITLE
Create a section for testnets and their purpose

### ISSUE THREAD

**Issue by evan-forbes** [celestiaorg/docs#1254]
We should formalize the purpose of the main longer lived testnets by documenting them. We all have social consensus around mocha being the network that is as close to mainnet as possible, but I don't think we have a formalized role for arabica. It could be useful to document to users that they should use mocha for longer term tests, and use arabica for bleeding edge things such as UX changes.

cc @cmwaters @Bidon15 

Discussions around the purpose of arabica are likely outside the scope this issue, however we should likely come to consensus over that before we can create docs and thus close this issue.

**Comment by jcstein** [comment]
existing pages for reference:

https://docs.celestia.org/nodes/participate
https://docs.celestia.org/nodes/mainnet
https://docs.celestia.org/nodes/arabica-devnet
https://docs.celestia.org/nodes/mocha-testnet

**Comment by Mackenzie-OO7** [comment]
@jcstein Hi, i'd like to work on this. Could you provide more details on what is expected in this section?

**Comment by jcstein** [comment]
gm @Mackenzie-OO7 - I'm actually fairly certain it's covered as an overview in https://docs.celestia.org/nodes/participate, but maybe the pages for mainnet/arabica/mocha could use a review of whether they speak to the differences in the networks and their individual purposes.

**Comment by Mackenzie-OO7** [comment]
> gm @Mackenzie-OO7 - I'm actually fairly certain it's covered as an overview in https://docs.celestia.org/nodes/participate, but maybe the pages for mainnet/arabica/mocha could use a review of whether they speak to the differences in the networks and their individual purposes.

Alright, thank you for clarifying. Let me take a look at them.

**Comment by jcstein** [comment]
cc’ing @rootulp for a second set of eyes here. i think maybe we can edit existing pages, but curious what you think

**Comment by rootulp** [comment]
I think the content on https://docs.celestia.org/nodes/participate addresses this issue. Specifically:

> A devnet focused on developers who want to deploy sovereign rollups on the latest changes from Celestia's codebase. Arabica will be updated frequently and might be unstable at times given new updates.

and

> A testnet focused on enabling validators to test out their infrastructure by running nodes connected to the network.

**Comment by jcstein** [comment]
agreed. i think we can close this issue, if anyone has reasons to reopen it, please do 😄
